charts/operator-wandb/values.yaml

```diff
@@ -81,9 +81,9 @@ global:
       # defaults to the otel-daemonset service
       host: ""
       # grpc default is 4317, http default is 4318
-      port: 4317
+      port: 4318
```

Comment:
Since this will break if they are not on a very recent version of the server image, I'm hesitant to make the change here and we should instead make the change here https://deploy.wandb.ai/v2/admin/channels/9205b19d-9103-4ef6-88e3-4d7a6eda61a7 when we are sure that the server version with the code to support this is deployed fleet wide